app/controllers/admin/application_controller.rb

```diff
@@ -15,10 +16,24 @@ def authenticate_admin
       # TODO Add authentication logic here.
     end
 
+    # Filter the content on the dashboard for one voting period at a time
+    def get_period
+      year = session[:year]
+      @period = VotingPeriod.where(year: year)[0]
+
+      # If there wasn't data in the session or it was an invalid year, default
+      # to the current year.
+      unless @period
```

Comment:
Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.